.circleci/config.yml

```diff
@@ -10,7 +9,7 @@ jobs:
           command: |
             apt-get update -y
             apt-get install cmake libfreetype6-dev libsdl2-dev -y
-            git submodule update --init
+            git submodule update --init    
```

Comment:
leading whitespace

Reply:
there went something very wrong, this file shouldn't be changed in the first place...